### DISCUSSION TITLE
Happy 25th to Vérité Quake, and thanks

### DISCUSSION THREAD

**Discussion by QUINTIX** [Novum/vkQuake#406] [General]
I can't believe there are no major articles out about this today. On 2-Dec-1996, over a month before the launch of GLQuake, the first ever hardware accelerated version of Quake was released. And unlike GLQuake, it was feature complete relative to the software version, include the underwater warping effect and dynamic lightmap textures.

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/0KxRXZhQuY8/0.jpg)](https://youtu.be/0KxRXZhQuY8?t=1285)

This is what I consider vkQuake a true successor to, more than GLQuake.
Given the [legal nonsense](https://twitter.com/libretro/status/1429038286538952706) Zenimax (and Microsoft by implication) dared to tie to the work of [Dimitris Giannakis](https://www.youtube.com/watch?v=33T7NPxnEKI), NightDive studios, Machine Games and id Software for the binary attached to re-release, the work done here is all the more valuable. Thank you for continuing to maintain this fork, and letting me be a tiny part of that work.

**Comment by garoto** [comment]
I remember stumbling upon the vQuake.zip file in some FTP and thinking the "v" in the "vQuake.exe" binary name was for the ViRGE chipset from S3. Was very disapointed after the download finished.
